build-packages/csa-generator-theme/index.js

```diff
@@ -7,7 +7,7 @@ const logger = require('@scandipwa/scandipwa-dev-utils/logger');
 const { getComposerDeps } = require('@scandipwa/scandipwa-dev-utils/composer');
 const writeJson = require('@scandipwa/scandipwa-dev-utils/write-json');
 
-const DEFAULT_PROXY = 'https://demo100-ors-1588667385-csa-hcx.scandipwa.cloud/';
+const DEFAULT_PROXY = 'https://tech-demo.scandipwa.com/';
```

Comment:
Put 40ksku, since tech demo has store codes, i.e. we would also need to add into core those store codes